add content files for glossary, add termes, change breakpoint display

dxb opened the pull request.

dxb committed:
add content files for glossary, add termes, change breakpoint display

tut-tuuut commented on src/glossaire.ts:
```diff
@@ -0,0 +1,62 @@
+  {
+    title: "Authentification forte",
+    link: "https://fr.wikipedia.org/wiki/Authentification_forte",
+    desc: `L'authentification forte repose sur des méthodes considérées comme plus sûr que la méthode mémoriel : le mote de passe unique à usage unique, le certficat numérique ou les données biométriques.`,
```
```suggestion
    desc: `L'authentification forte repose sur des méthodes considérées comme plus sûres que la méthode mémorielle : mot de passe à usage unique, certificat numérique ou données biométriques.`,
```

tut-tuuut commented on src/glossaire.ts:
```diff
@@ -0,0 +1,62 @@
+    title: "OTP",
+    link: "https://fr.wikipedia.org/wiki/Mot_de_passe_%C3%A0_usage_unique",
+    desc: `signifie « One–Time Password » en anglais, est un « Mot de passe à usage unique » comme son nom l'indique en français. 
+    Il est généré pour être utilisé et valable que lors d’une unique session.`,
```
```suggestion
    Il est généré pour être utilisé et valable lors d’une unique session.`,
```

tut-tuuut commented on src/glossaire.ts:
```diff
@@ -0,0 +1,62 @@
+  {
+    title: "France Connect",
+    link: "https://fr.wikipedia.org/wiki/FranceConnect",
+    desc: `Service d'identifiction et d'authentification mettant en relation différents fournisseurs d'identité pour accéder à différents fournisseurs de services.`,
```
```suggestion
    desc: `Service d'identification et d'authentification mettant en relation différents fournisseurs d'identité pour accéder à différents fournisseurs de services.`,
```

tut-tuuut commented on src/glossaire.ts:
```diff
@@ -0,0 +1,62 @@
+    title: "OpenID",
+    link: "https://fr.wikipedia.org/wiki/OpenID",
+    desc: `OpenID est un standard de communication qui permet à un utilisateur de s’authentifier auprès de plusieurs sites. 
+    Basé sur l'échange d'une preuve d'authenticité (un secret partagé) entre le serveur d'authentification (fournisseur d'identité) et les services tiers sur les quels l'utilisateurs souhaitent se connecter.`,
```
```suggestion
    Basé sur l'échange d'une preuve d'authenticité (un secret partagé) entre le serveur d'authentification (fournisseur d'identité) et les services tiers sur lesquels l'utilisateur souhaite se connecter.`,
```

tut-tuuut commented on src/glossaire.ts:
```diff
@@ -0,0 +1,62 @@
+  {
+    title: "Fournisseur de service",
+    link: "https://fr.wikipedia.org/wiki/OpenID#Acteurs",
+    desc: `Le forunisseur de service est l'application qui dexige une preuve de l’identité de l’utilisateur via un système fédéré d'authentification. Il délègue ainsi cette tache à ce dernier.`,
```
```suggestion
    desc: `Le forunisseur de service est l'application qui dexige une preuve de l’identité de l’utilisateur via un système fédéré d'authentification. Il délègue ainsi cette tâche à ce dernier.`,
```

tut-tuuut commented on src/glossaire.ts:
```diff
@@ -0,0 +1,62 @@
+    desc: `L'authentification forte repose sur des méthodes considérées comme plus sûr que la méthode mémoriel : le mote de passe unique à usage unique, le certficat numérique ou les données biométriques.`,
+  },
+  {
+    title: "Authentification à deux facteurs",
```
```suggestion
    title: "Double authentification",
```
Je change le titre puisque le contenu commence par "également appelée authentification à deux facteurs"

tut-tuuut commented on src/glossaire.ts:
```diff
@@ -0,0 +1,62 @@
+  {
+    title: "Authentification à deux facteurs",
+    link: "https://fr.wikipedia.org/wiki/Double_authentification",
+    desc: `Également appelé authentification à deux facteurs (2FA en anglais), cette authentification permet de sécuriser l'accès à un service en complétant l'accès par mot de passe traditionnel par une méthode dite d'authentification forte.`,
```
```suggestion
    desc: `Également appelée authentification à deux facteurs (2FA en anglais), cette authentification permet de sécuriser l'accès à un service en complétant l'accès par mot de passe traditionnel par une méthode dite d'authentification forte.`,
```

tut-tuuut reviewed (approved):
J'aime beaucoup l'idée d'extraire le contenu pour le séparer de la mise en forme. Je saurai faire la prochaine fois. J'ai ajouté quelques suggestions pour corriger les coquilles que j'ai vues.

dxb committed:
Update src/glossaire.ts

Co-authored-by: Agnès Haasser <1035145+tut-tuuut@users.noreply.github.com>

dxb committed:
Update src/glossaire.ts

Co-authored-by: Agnès Haasser <1035145+tut-tuuut@users.noreply.github.com>

dxb committed:
Update src/glossaire.ts

Co-authored-by: Agnès Haasser <1035145+tut-tuuut@users.noreply.github.com>

dxb committed:
Update src/glossaire.ts

Co-authored-by: Agnès Haasser <1035145+tut-tuuut@users.noreply.github.com>

dxb committed:
Update src/glossaire.ts

Co-authored-by: Agnès Haasser <1035145+tut-tuuut@users.noreply.github.com>

dxb committed:
Update src/glossaire.ts

Co-authored-by: Agnès Haasser <1035145+tut-tuuut@users.noreply.github.com>

dxb committed:
Update src/glossaire.ts

Co-authored-by: Agnès Haasser <1035145+tut-tuuut@users.noreply.github.com>

rebeccadumazert committed:
Update glossaire.ejs

Au cas où Matignon s'intéresse à Authentification entre deux réunions j'ai préféré ajouter un `fr-col-xs-12`pour que ce soit bien propre et lisible sur mobile !
